tools/local-sensor/main.go

```diff
@@ -88,6 +94,7 @@ func mustGetCommandLineArgs() localSensorConfig {
 	flag.BoolVar(&sensorConfig.Verbose, "verbose", sensorConfig.Verbose, "prints all messages to stdout as well as to the output file")
 	flag.DurationVar(&sensorConfig.Duration, "duration", sensorConfig.Duration, "duration that the scenario should run (leave it empty to run it without timeout)")
 	flag.StringVar(&sensorConfig.CentralOutput, "central-out", sensorConfig.CentralOutput, "file to store the events that would be sent to central")
+	flag.BoolVar(&sensorConfig.BinaryOutput, "binary", sensorConfig.BinaryOutput, "store all the events that would be sent to central in binary format")
```

Comment:
Maybe a better UI would be to allow a `-format` (or `-output-format`) param. The default would always be `json`, but you could provide `raw` or `bytes` and get the binary encoding.

Reply:
I like this. But check my other comment and let me know what you think about having a way of writing both formats at the same time.